docker/build_and_test.sh

```diff
@@ -1,25 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
possibly silly question, if we remove this file, what is doing the equivalent step of running and publishing the PACT files?

Reply:
https://github.com/alphagov/pay-jenkins-library/blob/master/vars/buildMultistageAppWithMetrics.groovy#L36-L42